roles/misc-perftools/tasks/main.yml

```diff
@@ -13,6 +13,20 @@
     - iotop
     - bwm-ng
 
+- name: Create directory for kvm metric collection configuration file
+  file: path=/opt/telegraf state=directory mode=0755
+  when: "'compute_hosts' in group_names"
+
+- name: Set configuration file for kvm metric collection
+  template:
+    src: /root/os-ansible-perftools/roles/misc-perftools/templates/kvm_virsh.py
```

Comment:
I used the copy module and it works as expected.  The example you reference uses the copy module.